keps/0000-kep-template.md

```diff
@@ -86,9 +83,24 @@ A table of contents is helpful for quickly jumping to sections of a KEP and for
 
 [Tools for generating]: https://github.com/ekalinin/github-markdown-toc
 
+**ACTION REQUIRED:** There must be an issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) referencing this KEP and targeting a release milestone **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)
+of the targeted release**.
+
+## Release Signoff Checklist
+
+Check these off as they are completed for the Release Team to track. These checklist items _must_ be updated for the enhancement to be released.
```

Comment:
How do we deal with differing release processes? For example, when kubectl is broken out into its own repo and has a release process and timing that is different from k8s/k8s?

Reply:
I'd prefer we deal with that later, and focus on getting this to an iterable place for kubernetes/kubernetes since that's the pressing need

Reply:
I agree to postpone that. It's worth specifying that this is for the main Kubernetes minor release, though.